lib/openstack_query/query_blocks/query_output.py

```diff
@@ -21,15 +20,38 @@ class QueryOutput:
     def __init__(self, prop_enum_cls: Type[PropEnum]):
         self._prop_enum_cls = prop_enum_cls
         self._props = set()
+        self._forwarded_outputs = {}
```

Comment:
We should have a type info for this, as it's not obvious what we map from / to without studying the setters